omod/src/main/webapp/resources/htmlforms/Walkin-Client-Data-Capture-tool.xml

```diff
@@ -0,0 +1,256 @@
+    
+                    disable_fields("other-regimen");                    
+                    disable_fields("reason-for-visit-other")                    
+
+                    jq("#art-regimen").change(function () {
+                        if (jq(this).find("select").find(":selected").val() == "90002") {
+                            enable_fields("other-regimen");
+                        } else {
+                            disable_fields("other-regimen");
+                        }
+                    });                 
+
+                    jq("#reason-for-visit").change(function () {
+                        if (jq(this).find("select").find(":selected").val() == "90002") {
+                            enable_fields("reason-for-visit-other");
+                        } else {
+                            disable_fields("reason-for-visit-other");
+                        }
+                    });
+    
+                    beforeSubmit.push(function () {
+                        var submitVal = true;
+
+                        var submitValStage = [];
+                        var submitValStageART = [];
+
+                        if(getValue('telephone-number.value')==""){
+                            jq().toastmessage('showErrorToast', "Telephone Number is required");
+                        }
+            
+                        if(jq("#patientNumber").find("input:first").val()==""){
+                            jq().toastmessage('showErrorToast', "Patient Clinic number missing");
+                        }
+
+                        if(getValue('reason-for-visit.value')==""){
+                            jq().toastmessage('showErrorToast', "Reason for visit is required");
+                        }
+            
+                        if(getValue('current-regimen.value')==""){
+                            jq().toastmessage('showErrorToast', "Current Regimen is required");
+                        }
+
+                        if(getValue('parent-health-facility.value')==""){
+                            jq().toastmessage('showErrorToast', "Parent Health Facility is required");
+                        }
+
+                        if(getValue('serial-number.value')==""){
+                            jq().toastmessage('showErrorToast', "Serial Number is required");
+                        }
+
+                        if(getValue('pt-clinic-number.value')==""){
+                            jq().toastmessage('showErrorToast', "PT Clinic Number is required");
+                        }
+
+                        if(getValue('tb-status.value')==""){
+                            jq().toastmessage('showErrorToast', "The TB Status is required");
+                        }
+
+                        if(getValue('no-of-art-pills.value')==""){
+                            jq().toastmessage('showErrorToast', "The Art Refill is required");
+                        }
+
+                        if (submitValStage.indexOf(false) != -1) {
```

Comment:
@ODORA0 This is not required, just add a variable which is set to false

Reply:
@ssmusoke Removed